<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaksutapaSivu</class>
 <widget class="QWidget" name="MaksutapaSivu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Maksutavat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../pic/pic.qrc">
        <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Tuloille</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="MaksutapaMuokkaus" name="tuloView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../pic/pic.qrc">
        <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Menoille</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="MaksutapaMuokkaus" name="menoView" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lisaGroup">
     <property name="title">
      <string>Näytä laskun lisätiedot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="laskuilleRadio">
        <property name="text">
         <string>Laskuille</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/lasku.png</normaloff>:/pic/lasku.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kaikilleRadio">
        <property name="text">
         <string>Kaikille</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/dokumentti.png</normaloff>:/pic/dokumentti.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="eiRadio">
        <property name="text">
         <string>Ei koskaan</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/sulje.png</normaloff>:/pic/sulje.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaksutapaMuokkaus</class>
   <extends>QWidget</extends>
   <header location="global">maaritys/maksutapamuokkaus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
